dateutil/tz/tz.py

```diff
@@ -1460,6 +1458,43 @@ def __get_gettz():
         tzlocal_classes += (tzwinlocal,)
 
     class GettzFunc(object):
+        """
+        gettz takes a variety of inputs and returns a time zone.
+
+        :param object:
+            A time zone name.
+            A ``TZ`` variable.
+            If not provided, the tzfile localtime is returned.
+
+        :return:
+            A :class:`dateutil.tz.tzfile`.
+
+        **Examples:**
+
+        You can get the local time.
+
+        ..code-block::
+
+            >>> frome dateutil.tz import gettz
+
+            >>> gettz()
+            tzfile('/etc/localtime')
+
+        You can parse time zone names.
+
+        ..code-block::
+
+            >>> gettz('Pacific/Kiritimati')
+            tzfile('/usr/share/zoneinfo/Pacific/Kiritimati')
+
+        You can parse a TZ variable.
+
+        ..code-block::
+
+            >>> from dateutil.tz import tzutc, UTC
+            >>> tgettz('AEST-10AEDT-11,M10.1.0/2,M4.1.0/3')
```

Comment:
The UTC and tzutc imports are unnecessary, and `tgettz` is definitely not right.